internal/policygen/schema.go

```diff
@@ -22,6 +22,11 @@ additionalProperties = false
 required = ["template_dir"]
 
 properties = {
+  version = {
+    description = "Optional constraint on the binary version required for this config."
```

Comment:
Given an example of the syntax of the constraint?

Reply:
Good point. The syntax is [here](https://www.terraform.io/docs/configuration/version-constraints.html), added links to it.